cmd/src/campaigns_common.go

```diff
@@ -265,7 +265,10 @@ func campaignsExecute(ctx context.Context, out *output.Output, svc *campaigns.Se
 
 	if len(specs) > 0 {
 		progress := out.Progress([]output.ProgressBar{
-			{Label: "Sending changeset specs", Max: float64(len(specs))},
+			{
+				Label: fmt.Sprintf("Sending %d changeset specs", len(specs)),
```

Comment:
This should probably have a special case to handle `len(specs) == 1` so we don't output `Sending 1 changeset specs`.

Reply:
Ah, good catch!